Word construction from a subset of letters.md

```diff
@@ -0,0 +1,31 @@
+# Summary
+This is a distractor task for the task "Hidden figure in a picture:" https://github.com/Watts-Lab/task-mapping/blob/fe161a548c3fe9ca5fec1749a2f98329f50a3ba1/tasks/Hidden%20figure%20in%20a%20picture.md
+
+Participants had to produce as many four letter words as possible given a set of nine letters.
+
+# References
+Main paper: https://psycnet-apa-org.proxy.library.upenn.edu/fulltext/2000-12254-014.pdf 
+
+# Stimuli
+## The visual components
+None mentioned
+
+# Procedure
+## Steps
+Each participant worked alone for 3 min on a distractor task in which they had to produce as many four letter words as possible from a given set of nine letters (FIND TASK)
```

Comment:
might be important to note that the subset of letters did not allow for the participants to create the target words from the Hidden figures experiment